src/faq

```diff
@@ -244,36 +215,36 @@
 
   <h4 id="living-standard">What does “Living Standard” mean?<a class="self-link" href="#living-standard"></a></h4>
 
-  <p>The WHATWG specifications are described as Living Standards. This means that they are standards
-  that are continuously updated as they receive feedback, either from Web designers, browser
+  <p>The WHATWG standards are described as Living Standards. This means that they are standards
+  that are continuously updated as they receive feedback, either from web designers, browser
```

Comment:
I suppose, although I was thinking of our general move from authors to developers.